_modules/ash_linux.py

```diff
@@ -261,8 +268,12 @@ def fips_enable():
         grub_args = _get_grub_args()
         if 'fips=1' not in grub_args:
             grubby_args.append('fips=1')
-        if 'boot=' not in grub_args:
-            grubby_args.append('boot={0}'.format(_get_boot_mount()))
+			
+        # Add boot= argument if a boot path exists.
+        boot_path = _get_boot_path()
+        boot_mount = _get_boot_mount(boot_path)
```

Comment:
Tiny nit, we don't _really_ need the boot_path assignment...

```suggestion
        boot_mount = _get_boot_mount(_get_boot_path())
```

Reply:
Done.